src/applications/vaos/components/PastAppointmentsList.jsx

```diff
@@ -72,7 +72,12 @@ export class PastAppointmentsList extends React.Component {
     } else if (pastStatus === FETCH_STATUS.succeeded && past?.length > 0) {
       content = (
         <>
-          <span className="vads-u-font-size--sm vads-u-display--block vads-u-margin-bottom--1">
+          <span
+            id="queryResultLabel"
+            tabIndex="-1"
```

Comment:
`focusElement` should add a tabIndex of -1 automatically, though I haven't verified that that works correctly here.

Reply:
I saw that too. I can remove it if you want.

Reply:
Yeah, I would go ahead and remove it.

Reply:
Done.